index.js

```diff
@@ -308,13 +310,16 @@ function updateTagsSpec(tags) {
 */
 function handleResponses(expressApp, 
   options = { 
-    swaggerUiServePath: 'api-docs', 
+    swaggerUiServePath: DEFAULT_SWAGGER_UI_SERVE_PATH, 
     specOutputPath: undefined, 
     predefinedSpec: {}, 
     writeIntervalMs: 1000 * 10, 
     mongooseModels: [], 
-    tags: undefined
+    tags: undefined,
+    ignoredNodeEnvironments: DEFAULT_IGNORE_NODE_ENVIRONMENTS,
   }) {
+
+  ignoredNodeEnvironments = options.ignoredNodeEnvironments || DEFAULT_IGNORE_NODE_ENVIRONMENTS;
```

Comment:
instead of handling `ignoredNodeEnvironments` here, why not do it inside the function's `options` parameters?

([here](https://github.com/mpashkovskiy/express-oas-generator/pull/69/files#diff-168726dbe96b3ce427e7fedce31bb0bcR319))

Reply:
Because the way it's implemented right now it's a default **complete** object. 
We need to do destructuring with `options` in the future.
If you supply any object right now, it overwrites the default one (completely).
So I need to handle both cases. Let me know if it's clear enough!

Reply:
Yes @kiprasmel I also was wondering why it is implemented that way, but failed to recall. Looks like it is common approach here and it is a bit ugly but let's leave it for another refactoring PR?

Reply:
Sure, I'm planning a nice refactoring PR after 1.x roadmap functionalities.